<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="db" uuid="dba7c76f-0936-4568-90ad-408e10373fef">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:D:\Django_Task\DjangoTask\db.sqlite3</jdbc-url>
    </data-source>
    <data-source source="LOCAL" name="db [2]" uuid="a1ce53e9-4f7c-4769-ba78-80edc7d90473">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:C:\Users\FUJITSU\Desktop\Django_Task\DjangoTask\db.sqlite3</jdbc-url>
    </data-source>
    <data-source source="LOCAL" name="Django default" uuid="576406ee-dff7-4975-a253-c5f46d497e1b">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/DjangoTask/settings.py</remarks>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:C:\Users\FUJITSU\Desktop\Django_Task\DjangoTask\db.sqlite3</jdbc-url>
    </data-source>
  </component>
</project>